Test transformers script

cvalarida opened the pull request:
## Description
Reworked an old script to test the bundle transformers. It could be cleaned up, but it works. If we end up needing to add anything else to it, we should refactor it to extract some functions and make it generally nicer to read.

## Testing done
Manually checked some things out.

## Screenshots
![image](https://user-images.githubusercontent.com/12970166/86853749-f86abd80-c06b-11ea-9bd8-210fa9150462.png)
![image](https://user-images.githubusercontent.com/12970166/86854098-a5ddd100-c06c-11ea-8423-c78e7f199da9.png)
![image](https://user-images.githubusercontent.com/12970166/86854198-d32a7f00-c06c-11ea-802a-72a7b31cf8c3.png)
![image](https://user-images.githubusercontent.com/12970166/86853834-26e89880-c06c-11ea-8247-fa29552f1cb2.png)

Commit:
Move test-cms-exports -> assemble-entity-tree

Commit:
Handle multiple entities from the cli

Commit:
Add command line usage and help

Commit:
Only read in the count

Commit:
Use entity instead of node

Commit:
Clean up the error output a bit

Commit:
Perform transformations on a certain bundle type

Commit:
Log entities transformed

Commit:
Entity at index

Commit:
assemble-entity-tree.js -> test-bundle-transformers.js

Commit:
.cv -> .js :facepalm:

Commit:
Color!

Commit:
Fiddling with the output

bkjohnson reviewed (approved):
The command line args are a nice addition!

The only comment is that we should probably update/remove the npm script that used that deleted file:

https://github.com/department-of-veterans-affairs/vets-website/blob/7e21ec671514631b3fd58d4fab5afa707ca25b73/package.json#L12

Commit:
Help output title change

Commit:
Remove build:content:test

We can just manually invoke the command; no need to clutter the NPM
scripts more.

Commit:
Remove extraneous eslint-disable-next-line no-console
